lib/themes/dosomething/paraneue_dosomething/includes/helpers.inc

```diff
@@ -360,13 +360,7 @@ function paraneue_dosomething_get_themed_reportback_item($data) {
     return NULL;
   }
 
-  if ($data->id && !$data->disable_reactions) {
-    $data->reaction_totals = [
-      'heart' => dosomething_kudos_get_total_for_file_by_term_name($data->id, 'heart'),
-    ];
-
-    $data->existing_kids = dosomething_kudos_get_kudos_for_user_by_file($data->id);
-  }
+  $data->kudos = dosomething_kudos_get_kudos_data_for_fid($data->id, $data->nid);
```

Comment:
You probs still need to work with an `if` conditional here if `$data->id` doesn't exist... there's no check or that in `dosomething_kudos_get_kudos_data_for_fid()`...